packages/perseus/src/util/parse-perseus-json/general-purpose-parsers/pipe-parsers.ts

```diff
@@ -1,13 +1,25 @@
 import {isFailure} from "../result";
 
 import type {
-    ParsedValue,
-    PartialParser,
     ParseContext,
+    ParsedValue,
     Parser,
+    PartialParser,
 } from "../parser-types";
 
-export function composeParsers<
+export function pipeParsers<T>(p: Parser<T>): ParserPipeline<T> {
```

Comment:
I am not familiar with these files, so forgive me if this is a newbie question. What is the purpose of adding this function and class to pipe-parsers for the Migration `answerArea` and removing unused fields. This looks like your cleaning up something else on top of this work, which isn't bad. But just curious how this loops back to the issue you're trying to fix.

Reply:
`pipeParsers` is used here: https://github.com/Khan/perseus/pull/1895/files/a5735edc9b887a5a9bd1ebde7df37f89f9ab72f1#diff-f4f46cb44d17e630e444aaecb0252590dd3274953230681bcaecfc81aab0df91R22

It's useful for composing more than two parsers in a readable way.